docs/user-guide/time-and-costs/cost-tracking/README.md

```diff
@@ -49,23 +51,21 @@ A new form is opened, where you can log unit costs. You can edit the following i
 
 The **spent unit costs** as well as the total amount of **overall costs** are displayed on the work packages details view.
 
-
-
-![Cost-tracking-overview](Cost-tracking-overview.png)
+![Bildschirmfoto 2022-02-12 um 18.51.29](/Users/jankeppler/Desktop/Bildschirmfoto 2022-02-12 um 18.51.29.png)
```

Comment:
Please rename the image file and change the name here, so there are no spaces in it.